jupyrdf/ipyradiant
# Release cleanup

Addressing comments left by sanbales on https://github.com/jupyrdf/ipyradiant/pull/82

Test procedure:
* CI passes
* Run doit locally if you want to be thorough (or check GraphExplorer example on binder). 

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/zwelz3/ipyradiant/release-cleanup?urlpath=lab/tree/index.ipynb)
